WireAnalytics/Tests/WireAnalyticsTests/AnalyticsServiceTests.swift

```diff
@@ -48,7 +46,6 @@ class AnalyticsServiceTests: XCTestCase {
     override func tearDown() {
         countly = nil
         sut = nil
-        super.tearDown()
```

Comment:
note: is it possible to add a swiftlint custom rule to get rid of super calls then?

Reply:
I don't know to be honest.
For now I can only offer to remove any superfluous calls we have at the moment.

Reply:
ok I understand, for now nothing to do then